cmd/localizer/list.go

```diff
@@ -44,12 +44,13 @@ func NewListCommand(_ logrus.FieldLogger) *cli.Command { //nolint:funlen
 			ctx, cancel := context.WithTimeout(c.Context, 30*time.Second)
 			defer cancel()
 
-			client, err := localizer.Connect(ctx, grpc.WithBlock(), grpc.WithInsecure())
+			client, closer, err := localizer.Connect(ctx, grpc.WithBlock(), grpc.WithInsecure())
 			if err != nil {
 				return errors.Wrap(err, "failed to connect to localizer daemon")
 			}
+			defer closer() //nolint:errcheck // Why: Nothing we can really do about the error that comes from closing the gRPC client connection.
```

Comment:
can you just have closer be a empty func or something if it returns an err?  i don't know what's actually idiomatic here, but it feels like it'd be safer to always do a defer closer() if possible.  but obvs you know go better than i do.

Reply:
yeah we can do that. the error is just from closing a grpc.ClientConn, so if that fails we can't do anything about it. I'll just eat the error in the closing function as opposed to having to nolint it everywhere